README.md

```diff
@@ -12,6 +12,29 @@ The commercial PyMOL product ("Incentive PyMOL") with maintenance and support is
 
 See [INSTALL](INSTALL).
 
+#### TL,DR;
+
+Clone repo and change to the directory
+```
+git clone https://github.com/schrodinger/pymol-open-source
+cd pymol-open-source/
+```
+
+Install with base dependencies
+```
+sudo apt update
+sudo apt install cmake netcdf-bin libnetcdf-dev libglm-dev libglew-dev libpng-dev libfreetype-dev libfreetype6 libfreetype6-dev
+pip install .
+```
+
+Install Qt5
+```
+sudo apt install qtcreator qtbase5-dev qt5-qmake cmake pyqt5-dev pyqt5-dev-tools python3-pyqt5
+pip install pyqt5
+pymol
+```
+![image](https://github.com/user-attachments/assets/b9388abb-d525-4ced-87fd-6396d1f44152)
```

Comment:
Is it okay if we remove this for now? I'm sure others on the team might have some opinions of what the window should look like here and if there should be one.